tests/unit/amo/components/TestInstallButtonWrapper.js

```diff
@@ -49,6 +42,9 @@ describe(__filename, () => {
       {
         // We need one more try than the default because of the withExperiment
         // HOC.
+        // Note that the wrapper has been removed, but will likely be replaced
+        // again for future experiments, so it makes sense to keep this in
+        // place.
```

Comment:
I don't know if that makes sense. If `maxTries` default value is too small, we could also increase it. More than 10 HOCs applied to a component seems like a lot, though.

Reply:
I guess increasing the default for `maxTries` is better than adding this comment. I agree that more than 10 HOCs seems like a lot, but looking at the debug info it's not necessarily distinct HOCs that affect this. As this is another example of keeping something around we won't need until the next experiment, I'll just remove it for now.